docs/src/references/tedge-binary-repository

```diff
@@ -0,0 +1,29 @@
+# Thin Edge Binary Repository
```

Comment:
I feel that "binary repository" is really misleading. This is not a repository more about exchanges and any file can be transferred not just binary ones.